samples/WebAPI/Program.cs

```diff
@@ -2,9 +2,9 @@
 // Licensed under the MIT License.
 
 using System.Text.Json.Serialization;
-using Microsoft.DurableTask;
 using Microsoft.DurableTask.Client;
 using Microsoft.DurableTask.Worker;
+using WebAPI;
```

Comment:
It is because I changed the namespace of `GeneratedDurableTaskExtensions` to be the project root namespace, and this file does not actually have a namespace (minimal APIs).

Reply:
Ah, I see that you updated the PR description with more context.

Reply:
Undid this change.